tests/trainer/test_dataloaders.py

```diff
@@ -234,22 +234,18 @@ def on_test_epoch_start(self, trainer, pl_module):
         self.test_epoch_count += 1
 
 
-@pytest.mark.parametrize(
-    ["limit_train_batches", "limit_val_batches", "limit_test_batches"], [(0.0, 0.0, 0.0), (1.0, 1.0, 1.0)]
```

Comment:
I haven't thoroughly reviewed yet but why are you removing all the tests that pass `0`?

Reply:
yes, because I have added a test that checks that dataloader isn't loaded with 0.